templates/zerver/emails/digest.source.html

```diff
@@ -54,48 +67,86 @@ <h2>Hot conversations</h2>
                         {% endfor %}
                     </div>
                 </div>
-                {% if convo.count > 0 %}<p>+ {{ convo.count }} more message{{ convo.count|pluralize }} by {{ convo.participants|display_list(4) }}.</p>{% endif %}
+                {% if convo.count > 0 %}
+                <p>
+                    {% trans count=convo.count, participants=convo.participants|display_list(4) %}
+                    + {{ count }} more message by {{ participants }}.
+                    {% pluralize count %}
+                    + {{ count }} more messages by {{ participants }}.
+                    {% endtrans %}
+                </p>
+                {% endif %}
             {% endfor %}
         </div>
         {% endfor %}
-        <p><a href="{{ realm_uri }}">Catch up on the rest of these conversations.</a></p>
+        <p><a href="{{ realm_uri }}">{{ _("Catch up on the rest of these conversations.") }}</a></p>
 
     {% endif %}
 
     {% if new_users and new_streams.html %}
     <br />
-    <h2>Group updates</h2>
+    <h2>{{ _("Group updates") }}</h2>
     {% elif new_users %}
     <br />
-    <h2>New users</h2>
+    <h2>{{ _("New users") }}</h2>
     {% elif new_streams.html %}
     <br />
-    <h2>New streams</h2>
+    <h2>{{ _("New streams") }}</h2>
     {% endif %}
 
     {% if new_streams.html %}
-    <p>{% if new_stream_count and new_stream_count > 1 %}A new stream was{% else %}Some new streams were{% endif %} created:</p>
+    <p>
+        {% if new_stream_count and new_stream_count > 1 %}
+            {% trans %}
+            A new stream was created:
```

Comment:
@timabbott Shouldn't this be "Some new streams were created"? Also, there was no variable called new_stream_count. It was new_stream**s**_count. I added a commit to fix that in the PR. 4da2db3da27541cfa0d9a7480ce27ed80fee00ed